geopandas/array.py

```diff
@@ -375,14 +389,16 @@ def __setitem__(self, key, value):
 
     if compat.USE_PYGEOS:
 
-        def __getstate__(self):
-            return (pygeos.to_wkb(self.data), self._crs)
+        if compat.USE_PYGEOS:
+
+            def __getstate__(self):
+                return (pygeos.to_wkb(self.data), self._crs)
 
-        def __setstate__(self, state):
-            geoms = pygeos.from_wkb(state[0])
-            self._crs = state[1]
-            self.data = geoms
-            self.base = None
+            def __setstate__(self, state):
+                geoms = pygeos.from_wkb(state[0])
+                self._crs = state[1]
+                self.data = geoms
+                self.base = None
```

Comment:
I think you mistakenly duplicated the `if compat.USE_PYGEOS:` line here. Seems than all changes on these lines can be reverted back.

Reply:
Yup, reverted.